kubelet/changelog.d/16821.added

```diff
@@ -0,0 +1 @@
+add startup latency metrics
```

Comment:
Thanks for the contribution @chenfeilee!

Please turn this changelog entry into a full english sentence. Imagine it will be read by a user of the integration: what would they care about? What would given them enough context to understand the change?